pymatgen/io/lammps/inputs.py

```diff
@@ -31,11 +32,111 @@
 
 class LammpsInputFile(InputFile):
     """
-    Class representing a LAMMPS input settings file, e.g. in.lammps
+    Class representing a LAMMPS input settings file, e.g. in.lammps.
     """
 
-    pass
+    def __init__(self, input_settings: OrderedDict = None):
+        self.input_settings = input_settings
+
+    def get_string(self):
+        """
+        Generates and returns the string representation of LAMMPS input file.
+
+        Returns: String representation of LAMMPS input file.
+
+        """
+
+        lammps_input = '## LAMMPS input generated from LammpsInputConstructor\n\n'
+        for stage, command_dict in self.input_settings.items():
+            for command, args in command_dict.items():
+                if command.startswith('comment') or command.startswith('header'):
+                    if command.startswith('header'):
+                        lammps_input += '\n'
+                    lammps_input += self.add_string(args)
+                else:
+                    lammps_input += self.format_command(command, args)
+        return lammps_input
+
+    @staticmethod
+    def format_command(command, args):
+        """
+        Simple format helper for adding command with the arguments.
+
+        Args:
+            command: LAMMPS command.
+            args: Arguments for LAMMPS command.
+
+        Returns:
+
+        """
+
+        return "{0:20} {1}\n".format(command, args)
+
+    @staticmethod
+    def add_string(string):
+        """
+        Helper method for adding string to lammps input file.
+        Args:
+            string: String to be added to lammps input file.
+        """
+        # return '\n' + string + '\n'
+        return string + '\n'
+
```

Comment:
It's odd to me that `add_string` is in `LammpsInputFile` while `add_command` and `add_comment` are in `LammpsInputConstructor`. If you are able to merge the two classes this will be more consistent (although it's not clear to me why you need `add_string` if you have the other two)